sdkv2.0.0/sdkv2.0.0/MojioSDK/App.swift

```diff
@@ -27,6 +27,10 @@ class App: Object, Mappable {
         self.init();
     }
     
+    override static func primaryKey() -> String? {
+        return "Id"
```

Comment:
Maybe a static final constant somewhere for this "Id" string?

In the Android SDK what I've done is make the top-level models extend "MojioObject" which defined the Id field (along with CreatedOn, LastModified, and Links). Maybe that doesn't apply here because this function is static? (which seems weird to me but I assume this is a Swift/Realm thing).